pantheon_advanced_page_cache.module

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\image\Entity\ImageStyle;
```

Comment:
Is `image` (lowercase `i`) correct here?

Reply:
yes, of course. :)   why wouldn't it be...
https://api.drupal.org/api/drupal/core%21modules%21image%21src%21Entity%21ImageStyle.php/8.2.x